app/config/config.yml

```diff
@@ -21,7 +21,7 @@ services:
     PrestaShopBundle\Controller\:
         resource: "%kernel.root_dir%/../src/PrestaShopBundle/Controller/*"
         exclude: "%kernel.root_dir%/../src/PrestaShopBundle/Controller/Api"
-        tags: ['prestashop.core.controllers']
+        tags: ['prestashop.core.controllers', 'controller.service_arguments']
```

Comment:
Unrelated to this PR, but I saw that Symfony default services configuration has this. So I add it I think it's needed.

Reply:
for information => https://symfony.com/doc/3.4/service_container/3.3-di-changes.html#controllers-are-registered-as-services

Reply:
Thanks, so it's to enable autowiring arguments for controller action methods

Reply:
yes, at least according to the documentation: I can't be sure it works in your case, maybe a micro module with autowired args in a controller maybe be proposed ? :)

Reply:
This PR will be closed as a better solution has be found. Tag will be added in https://github.com/PrestaShop/PrestaShop/pull/18845